query.ts

```diff
@@ -188,6 +188,22 @@ export class Query {
     return this;
   }
 
+  clone() {
+    const c = new Query();
```

Comment:
Use more meaningful and clear variable names

Reply:
`newQuery`, `newBuilder`, `clonedQuery` or ` clonedBuilder`. Which one do you think is better? or any other suggestions?  
thank you for your review!

Reply:
updated! @manyuanrong